### ISSUE TITLE
Performance Metrics - Step 5

### ISSUE THREAD

**Issue by IKEA0x90** [evil-cry/cyber-analytics-ml#20]
Accuracy is evaluated for a Binary Classifier of benign vs. malware. Measuring OvO and OvA is not necessary. (Jaxiom)
A normalized confusion matrix of the multi-class accuracy for the malware families. (IKEA)

**Comment by IKEA0x90** [comment]
Sitting at a 50% accuracy. Need to investigate whether this is an issue with hyperparameters or the code.

**Comment by evil-cry** [comment]
1. Evaluate potential overfitting as training data has a high accuracy but testing data has a very low accuracy
2. Determine if class imbalance affects bias towards a class
3. Determine if the model is sensitive to the scale of features
